Celeste.Mod.mm/Patches/MapData.cs

```diff
@@ -175,8 +180,35 @@ public patch_MapData(AreaKey area)
             AreaData area = AreaData.Get(Area);
             AreaMode mode = Area.Mode;
 
+            MapMeta metaParsedFromFile = null;
```

Comment:
This is the core of the PR's logic. This should, ideally, be the _only_ place we pull a MapMeta out of the everest asset store: in the middle of MapData parsing.

Reply:
In service of this, the method that calls this method was patched to call this method even if the map has no meta node in its packed data, so meta may be null here.